ui/app/components/app/modals/deposit-ether-modal/deposit-ether-modal.component.js

```diff
@@ -141,7 +142,16 @@ export default class DepositEtherModal extends Component {
               title: this.context.t('buyWithWyre'),
               text: this.context.t('buyWithWyreDescription'),
               buttonLabel: this.context.t('continueToWyre'),
-              onButtonClick: () => toWyre(address),
+              onButtonClick: () => {
+                this.context.metricsEvent({
+                  eventOpts: {
+                    category: 'Onboarding',
```

Comment:
Does the `Onboarding` category make sense here? 🤔

Reply:
It's primarily an way to _onboard_ people

Reply:
Not really? It's a way to buy Eth. Existing users need to buy Eth too. It's not even presented during onboarding.

Reply:
Updated to use "Accounts"

Reply:
Let me know if you have a different suggestion

Reply:
Maybe "On-Ramp"? Or "Integrations", or "Third-Party", or "Exchange"? 🤷‍♂️ 
A new category would just have this one action type in it, but that's not unprecedented. We'll likely have more related metrics soon as well.

I guess "Accounts" is fine too.